.gitmodules

```diff
@@ -49,3 +49,6 @@
 	shallow = true
 	path = External/robin-map
 	url = https://github.com/Tessil/robin-map.git
+[submodule "External/Vulkan-Headers"]
```

Comment:
How about `shallow = true` for sane checkout times?

Reply:
Made shallow.